schemas/asset_schema.json

```diff
@@ -29,7 +29,7 @@
                 },
                 "positionZ": {
                     "type": "number",
-                    "description": "Z-position of the axle with respect to the vehicles center of the bounding box projected to the ground. Usually this is half of wheel diameter. Unit:meter; Range:[0..inf[.",
+                    "description": "z-position of the axle with respect to the vehicles center of the bounding box projected to the ground. Usually this is half of wheel diameter. Unit:meter; Range:[0..inf[.",
```

Comment:
Why lower case at the beginning of a sentence?

Reply:
It would not be consistent with the rest of the text to write 'z-position' with an uppercase letter, even at the beginning of a sentence. Therefore, I have added 'the' at the beginning, making it 'The z-position of the...'."